third_party/libuv_test.patch

```diff
@@ -0,0 +1,134 @@
+diff --git a/test/test-getsockname.c b/test/test-getsockname.c
+index 565c17fe..e278b7d4 100644
+--- a/test/test-getsockname.c
++++ b/test/test-getsockname.c
+@@ -30,8 +30,9 @@ static const int server_port = TEST_PORT;
+ /* Will be updated right after making the uv_connect_call */
+ static int connect_port = -1;
+ 
+-static int getsocknamecount = 0;
++static int getsocknamecount_tcp = 0;
```

Comment:
I wonder why we need to separate the callback counting here (thus this patch), is there a difference on iOS test context that makes the vanilla libuv test invalid here ?

Reply:
The static variable was used by multi test cases, it was OK since other languages use fork and count separately. But in iOS test, we have to run all tests in the same process, so we need separate variables to count separately.